docker image arm64 compatibility

**What happened**:
the following docker images currently do not support arm64:
[ERROR] registry.cn-beijing.aliyuncs.com/yunionio/baremetal-agent:v3.7.5
[ERROR] registry.cn-beijing.aliyuncs.com/yunionio/climc:v3.7.5
[ERROR] registry.cn-beijing.aliyuncs.com/yunionio/web-ee:v3.7.5

please update the base images.

--- a/Makefile
+++ b/Makefile
@@ -1,0 +1,9 @@
+ROOT_DIR := $(CURDIR)
+
+REGISTRY ?= "registry.cn-beijing.aliyuncs.com/yunionio"
+
+VERSION ?= $(shell git describe --exact-match 2> /dev/null || \
+                git describe --match=$(git rev-parse --short=8 HEAD) --always --dirty --abbrev=8)
+
+image:
+	DEBUG=$(DEBUG) ARCH=$(ARCH) TAG=$(VERSION) REGISTRY=$(REGISTRY) $(ROOT_DIR)/scripts/docker-push.sh

